okhttp/src/main/java/okhttp3/OkHttpClient.java

```diff
@@ -714,6 +717,11 @@ public Builder addNetworkInterceptor(Interceptor interceptor) {
       return this;
     }
 
+    public Builder x509TrustManager(X509TrustManager x509TrustManager) {
```

Comment:
Not sure how to test this properly, looks like it'll require mock a lot of objects